workshop/7-PerformanceOptimization/7.1-Layer0.ipynb

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
Is this notebook complete? 

It would be great to have some overview and introductory content on what XDN is, why it's great, how it works, and so on. Workshop attendees are looking to be educated.

Also, we were going to add a separate notebook in the Experimentation folder for doing edge testing with XDN?